hackathon_site/dashboard/frontend/src/api/types.ts

```diff
@@ -99,10 +99,33 @@ export interface Order {
     team_id: number;
     team_code: string;
     status: OrderStatus;
+    request: {
+        id: number;
+        requested_quantity: number;
+    }[];
     created_at: string;
     updated_at: string;
 }
 
+/** Sanitized Orders */
```

Comment:
'Sanitized" meaning that it will remove the unnecessary attributes correct?

Reply:
Sanitized in this case means your taking one object struction and changing it to another to fit your needs, so it's not "removing" unneccesary attributes but reorganizing them or using them to calculate other attributes.